BraintreeModels/build.gradle

```diff
@@ -34,28 +34,6 @@ def playServicesWallet = "com.google.android.gms:play-services-wallet:${rootProj
 
 dependencies {
     implementation project(':SharedUtils')
-
```

Comment:
Should this gradle file and the `BraintreeModels` directory it's in be deleted?

Reply:
Yeah definitely good catch. We used an Android studio wizard to delete the module I guess it skips the files lol.